com.unity.multiplayer.mlapi/Tests/Runtime/Transport/SIPTransport.cs

```diff
@@ -229,7 +229,7 @@ public override void Send(ulong clientId, ArraySegment<byte> data, NetworkChanne
             // Create copy since MLAPI wants the byte array back straight after the method call.
             // Hard on GC.
             byte[] copy = new byte[data.Count];
-            Buffer.BlockCopy(data.Array, data.Offset, copy, 0, data.Count);
+             Buffer.BlockCopy(data.Array, data.Offset, copy, 0, data.Count);
```

Comment:
```suggestion
            Buffer.BlockCopy(data.Array, data.Offset, copy, 0, data.Count);
```

Reply:
No clue where this space came from. Wtf.

Reply:
Done.